libs/model-ad/api-description/openapi/openapi.yaml

```diff
@@ -491,6 +517,67 @@ components:
         - center
         - modified_genes
         - available_data
+    CorrelationResult:
+      type: object
+      description: Correlation result
+      properties:
+        correlation:
+          type: number
+          description: Correlation value
+        adj_p_value:
+          type: number
+          description: Adjusted p-value
+      required:
+        - correlation
+        - adj_p_value
+    DiseaseCorrelation:
+      type: object
+      description: Disease Correlation
+      properties:
+        name:
+          type: string
+          description: Model name
+        matched_control:
+          type: string
+          description: Matched control for the model
+        model_type:
+          type: string
+          description: Type of model
+        modified_genes:
+          type: array
+          items:
+            type: string
+          description: List of modified genes
+        cluster:
+          type: string
+          description: Cluster information
+        age:
+          type: string
+          description: Age of the model
+        sex:
+          $ref: '#/components/schemas/Sex'
+        IFG:
+          $ref: '#/components/schemas/CorrelationResult'
+        PHG:
+          $ref: '#/components/schemas/CorrelationResult'
+        TCX:
+          $ref: '#/components/schemas/CorrelationResult'
+        CBE:
+          $ref: '#/components/schemas/CorrelationResult'
+        DLPFC:
+          $ref: '#/components/schemas/CorrelationResult'
+        FP:
+          $ref: '#/components/schemas/CorrelationResult'
+        STG:
+          $ref: '#/components/schemas/CorrelationResult'
```

Comment:
The brain-region correlation results are written in uppercase acronyms (IFG, PHG, TCX, CBE, DLPFC, FP, STG).
For consistency, these should be converted to lowercase snake_case.

Reply:
Current | Suggested snake_case
-- | --
IFG | ifg
PHG | phg
TCX | tcx
CBE | cbe
DLPFC | dlpfc
FP | fp
STG | stg

Reply:
I can approve this PR as is regarding this comment but there should be a follow-up discussion / ticket to address this comment.